templates/logchimp/meta.yaml

```diff
@@ -0,0 +1,39 @@
+name: LogChimp
+description: Build better products with customer feedback.
+instructions: null;
```

Comment:
I'm not sure if this is a correct value for instructions. Leave it empty if there are no instructions.

Reply:
Whoops, sorry. Put that there by accident!